common/xbps-src/shutils/common.sh

```diff
@@ -250,6 +250,19 @@ source_file() {
     fi
 }
 
+into_array() {
```

Comment:
Compare this with the much simpler implementation at https://github.com/archlinux/mkinitcpio/blob/master/functions#L299

Reply:
That `@a` trick is good, else one also could use `case` with a pattern.

Reply:
bash never fails to surprise me with weird features like this. I've simplified my implementation to use this